charts/observability-pipelines-worker/values.yaml

```diff
@@ -33,6 +33,7 @@ datadog:
   ## Set to 'us3.datadoghq.com' to send data to the US3 site.
   ## Set to 'us5.datadoghq.com' to send data to the US5 site.
   ## Set to 'ddog-gov.com' to send data to the US1-FED site.
+  ## Set to 'ap1.datadoghq.com' to send data to the AP1 site.
```

Comment:
if a file is modified in a chart like here in `observability-pipelines-worker`, it is mandatory to also bump the chart version and add an entry in the corresponding changelog.
If the chart version is not updated, the new chart version and documentation will not be created.